### ISSUE TITLE
My Singing Monsters [PCSE00459]

### ISSUE THREAD

**Issue by LennyFaze0** [Vita3K/compatibility#402]
# App summary
- App name: My Singing Monsters
- App serial: [PCSE00459]
- App version: 1.0

# Vita3K summary
- Version: v0.1
- Build number: 1419
- Commit hash: https://github.com/Vita3K/Vita3K/commit/2bf8df78d46028cba4de4d9e76a9afeab052a47a 

# Test environment summary
- Tested by: LennyFaze
- OS: Windows 10
- CPU: AMD RYZEN 1500X
- GPU: NVIDIA RTX 2060 SUPER
- RAM: 16 GB

# Issues

Unimplemented sqlite3_exec import called.
Unimplemented sqlite3_free import called.
Unimplemented sceRtcGetAccumulativeTime import called.
Unimplemented sceAppUtilLoadSafeMemory import called.
Unimplemented sceAppUtilSaveSafeMemory import called.
Unicorn error 0x6. Invalid memory read (UC_ERR_READ_UNMAPPED)
Thread PCSE00459 experienced a unicorn error.
# Screenshots
![](https://?)
![Screenshot (44)](https://user-images.githubusercontent.com/65937293/82861790-71aab880-9ee3-11ea-9b04-11b6d10a1f5c.png)
The loading screen appears, disappears, reappears, then it crashes

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/4679964/vita3k.log)
# Recommended labels

- Bootable
- crash

**Comment by benderscruffy** [comment]
Vita3K v0.1.5 2688-6ec6eddd
stuck on the loading screen
[vita3k.log](https://github.com/Vita3K/compatibility/files/8956332/vita3k.log)
![Vita3K exe 22_06_2022 5_04_59 PM](https://user-images.githubusercontent.com/2696404/174995894-cb4a8bf1-acec-4e60-a509-f7e638fc576d.png)
![Vita3K v0 1 5 2688-6ec6eddd _ My Singing Monsters (PCSE00459) _ 150 ms_frame (7 frames_sec) _ 960x544  22_06_2022 5_03_41 PM](https://user-images.githubusercontent.com/2696404/174995903-5d1ae9fe-048b-4c93-8f44-534216987552.png)

**Comment by LIBBYXDPG3D** [comment]
sorry im new to github how do i download the zip file i dont see one. any advice would help!!!

**Comment by LIBBYXDPG3D** [comment]
oh nvm i found out why there's no download

**Comment by CrazyCobraCC** [comment]
Is there any chance this will ever be supported?

**Comment by Imadeanaccountjustpostthis** [comment]
> Is there any chance this will ever be supported?

same question, I'm really curious to see this version

**Comment by CupLukeKun** [comment]
> > Is there any chance this will ever be supported?
> 
> same question, I'm really curious to see this version
Me too because I like the old UI textures better than the current ones I wish MSM could release an option where you can switch to the old UI Textures.

**Comment by CrazyCobraCC** [comment]
Using my M1 MacBook Pro, Vulcan is sadly the only option at the moment, and it's honestly gotten worse since last time i tried it. The loading arrow used to appear all weird like always, but now the arrow isnt rendering anymore, just whatever this is. It crashes right after loading into whatever this is supposed to be...

<img width="946" height="528" alt="Image" src="https://github.com/user-attachments/assets/3935f368-7e96-4762-b4ce-00994671d95d" />
<img width="949" height="534" alt="Image" src="https://github.com/user-attachments/assets/a4d944b1-ad0a-48e1-868b-f4bb06cfffe3" />

**Comment by Clicktuck-Suskriberz** [comment]
<img width="960" height="543" alt="Image" src="https://github.com/user-attachments/assets/c7b1395c-11bf-4c3b-983e-e6704c7c50f4" />
I am in a new layer of hell. I am missing multiple decorations, Toe Jammer, Fwog, Bowgart, Maw, and Pummel. The background barely functions. Grennich is glad he can't be woken up here and remains eternally asleep because otherwise he surely would've suffered from epileptic shock by now.

**Comment by CupLukeKun** [comment]
> <img alt="Image" width="960" height="543" src="https://private-user-images.githubusercontent.com/120124607/593646353-c7b1395c-11bf-4c3b-983e-e6704c7c50f4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzkxMjY3MzIsIm5iZiI6MTc3OTEyNjQzMiwicGF0aCI6Ii8xMjAxMjQ2MDcvNTkzNjQ2MzUzLWM3YjEzOTVjLTExYmYtNGMzYi05ODNlLWU2NzA0YzdjNTBmNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDUxOFQxNzQ3MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YWRjZGM4ZGY3ZjI4ZDQ1ODYwOWQzNjA4ZDQyNjBhNDVkNWY1YmE3YjM2YjAxOTRmZDI5MWNmYTk2OTA5ODE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.zW5UWfI8cFEz7DI85wjget1KvSb4yk6xvF2gxUgeaGo"> I am in a new layer of hell. I am missing multiple decorations, Toe Jammer, Fwog, Bowgart, Maw, and Pummel. The background barely functions. Grennich is glad he can't be woken up here and remains eternally asleep because otherwise he surely would've suffered from epileptic shock by now.

Yeah, it was really bad, you can't see the eggs in the breeding structure and the nursery which made it a bit worse, I still have an old version of Vita 3K before this new visual glitch was added and you could see the eggs in the nursery and Breeding structure and there were no or a lack of shader corrupt boxes.

**Comment by Clicktuck-Suskriberz** [comment]
btw this is on OpenGL. I can't even open it on Vulkan. The **entire** emulator crashes.

**Comment by CupLukeKun** [comment]
When I open the game on Vulkan, it doesn't crash for me, I used the latest version of Vita 3K to test that for myself.

**Comment by dakktzill** [comment]
something i've noticed is that when images are .pvr, they have a higher chance of working, so i think the main rendering issues is w pngs
